### PR TITLE
Create multi_metadata.yaml for graphite exporters

### PR DESCRIPTION
**Pull request by shyamvalsan**
##### Additional Information

<details> <summary>For users: How does this change affect me?</summary>
  
</details>

### CODE REVIEW

File: exporting/graphite/multi_metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+          - name: 'destination'
+            default_value: 'no'
+            description: 'Accepts a space separated list of hostnames, IPs (IPv4 and IPv6) and ports to connect to. Netdata will use the first available to send the metrics.'
+            required: true
+            detailed_description: |
+              The format of each item in this list, is: [PROTOCOL:]IP[:PORT].
+              - PROTOCOL can be udp or tcp. tcp is the default and only supported by the current exporting engine.
+              - IP can be XX.XX.XX.XX (IPv4), or [XX:XX...XX:XX] (IPv6). For IPv6 you can to enclose the IP in [] to separate it from the port.
+              - PORT can be a number of a service name. If omitted, the default port for the exporting connector will be used.
+
```
Comment (thiagoftsm) [inline exporting/graphite/multi_metadata.yaml:49]: Hello @shyamvalsan ,

The same issue here, we have warnings from `yamllint`.

File: exporting/graphite/multi_metadata.yaml [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+            required: true
+            detailed_description: |
+              The format of each item in this list, is: [PROTOCOL:]IP[:PORT].
+              - PROTOCOL can be udp or tcp. tcp is the default and only supported by the current exporting engine.
+              - IP can be XX.XX.XX.XX (IPv4), or [XX:XX...XX:XX] (IPv6). For IPv6 you can to enclose the IP in [] to separate it from the port.
+              - PORT can be a number of a service name. If omitted, the default port for the exporting connector will be used.
+  
+              Example IPv4:
+                ```yaml
+                destination = 10.11.14.2:4242 10.11.14.3:4242 10.11.14.4:4242
```
Comment (thiagoftsm) [inline exporting/graphite/multi_metadata.yaml:52]: Please, modify default `OpenTSDB` port to `2003`.

File: exporting/graphite/multi_metadata.yaml [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+          - name: 'data source'
+            default_value: ''
+            description: 'Selects the kind of data that will be sent to the external database. (as collected|average|sum)'
+            required: false
+          - name: 'hostname'
+            default_value: '[global].hostname'
+            description: 'The hostname to be used for sending data to the external database server.'
+            required: false
+          - name: 'prefix'
+            default_value: 'Netdata'
```
Comment (thiagoftsm) [inline exporting/graphite/multi_metadata.yaml:76]: The default prefix is lowercase.

File: exporting/graphite/multi_metadata.yaml [hunk 4]
```diff
@@ -0,0 +1,212 @@
...
+              send data to the same database. This randomness does not affect the quality of the data, only the time they are sent.
+          - name: 'buffer on failures'
+            default_value: '10'
+            description: |
+              The number of iterations (`update every` seconds) to buffer data, when the external database server is not available. 
+            required: false
+            detailed_description: |
+              If the server fails to receive the data after that many failures, data loss on the connector instance is expected (Netdata will also log it).
+          - name: 'timeout ms'
+            default_value: '2 * update_every * 1000'
```
Comment (thiagoftsm) [inline exporting/graphite/multi_metadata.yaml:95]: Like I explained in the previous one, let us set default value `20000` and explain details in `description`.

File: exporting/graphite/multi_metadata.yaml [hunk 5]
```diff
@@ -0,0 +1,212 @@
...
+          - name: 'Detailed Configuration for a remote, secure host'
+            folding:
+              enabled: false
+            description: 'Add `:https` modifier to the connector type if you need to use the TLS/SSL protocol. For example: `remote_write:https:my_instance`.'
+            config: |
+              [graphite:https:netdata]
+                  enabled = yes
+                  username = my_username
+                  password = my_password
+                  destination = remote_host_url:2003
```
Comment (thiagoftsm) [inline exporting/graphite/multi_metadata.yaml:168]: Please, modify `destination` and fill it wit an IP or `localhost`.

**Review by sashwathn — DISMISSED** [review]

**Review by sashwathn — APPROVED** [review]

**Review by thiagoftsm — APPROVED** [review]

**Pull request merged**